test/terra/backends/aer_simulator/test_options.py

```diff
@@ -145,3 +146,28 @@ def test_shots_run_option(self, method, device):
         result = backend.run(qc, shots=shots).result()
         value = sum(result.get_counts().values())
         self.assertEqual(value, shots)
+
+    def test_mps_approximation(self):
+        """Test MPS approximation"""
+        shots = 4000
+        method="matrix_product_state"
+        backend_exact = self.backend(method=method)
+        backend_approx = self.backend(method=method,
+                                     matrix_product_state_max_bond_dimension=8)
+        # The test must be large enough and entangled enough so that
+        # approximation actually does something
+        n = 10
+        circuit = QuantumCircuit(n, n)
```

Comment:
You don't need clbits anymore (change to `QuantumCircuit(n)` )

Reply:
okay.